Foundation/NSStream.swift

```diff
@@ -169,53 +169,53 @@ public protocol NSStreamDelegate : class {
 // NSString constants for the propertyForKey/setProperty:forKey: API
 // String constants for the setting of the socket security level.
 // use this as the key for setting one of the following values for the security level of the target stream.
-public let NSStreamSocketSecurityLevelKey: String = "" // NSUnimplemented
+public let NSStreamSocketSecurityLevelKey: String = "NSStreamSocketSecurityLevelKey"
```

Comment:
technically the value is kCFStreamPropertySocketSecurityLevel
since this will need to interop with CF code. once NSStream is done.